lib/please_cli/please_cli/decision_task.py

```diff
@@ -323,6 +344,197 @@ def get_task(task_group_id,
     }
 
 
+NixPath = str
+NixHash = str
+NixHashes = typing.List[typing.Tuple[NixPath, NixHash]]
```

Comment:
You introduce some new complexity by using `typing` in this same PR: is it necessary ?